mandatory/2-function-creation.js

```diff
@@ -16,7 +19,14 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  let x = Math.round(num * 100) / 100;
```

Comment:
This works wonders, good use of Math.round() 👍 

An alternative would be to use `parseFloat(num.toFixed(2))`
 - **parseFloat()** - Takes an input and converts it into a float
 - **num.toFixed(2)** - Rounds the new `num` float up to 2 decimal places.
 
 Just to get you thinking a little, if my number is more than 100, do I need the function to run the Math logic on that number if we're just going to return the string "100%"? From a performance point of view, it's extremely inconsequential, but perhaps moving the `x` definition above the `return` line could prevent it being executed on numbers greater than 100?
 
 I would probably make `x` a `const` as well since its value is never reassigned.